src/kubernetes_api_objects/error.rs

```diff
@@ -16,4 +16,10 @@ pub enum APIError {
     Other
 }
 
+pub enum ParseDynamicObjectError {
+    MissingField,
+    UnexpectedType,
+    Error,
```

Comment:
Better to have a more concrete name than `Error`

Reply:
It is a general one because we don’t know the concrete error info of exec code.